app/src/main/java/nerd/tuxmobil/fahrplan/congress/models/DateInfos.kt

```diff
@@ -0,0 +1,56 @@
+package nerd.tuxmobil.fahrplan.congress.models
+
+import info.metadude.android.eventfahrplan.commons.temporal.Moment
+import info.metadude.android.eventfahrplan.commons.temporal.Moment.Companion.now
+
+import java.util.ArrayList
+
+class DateInfos : ArrayList<DateInfo>() {
+    fun sameDay(timestamp: Moment, sessionListDay: Int): Boolean {
+        val currentDate = timestamp.startOfDay()
+        return this.find {
+            (dayIndex, date) ->
+                dayIndex == sessionListDay && date == currentDate
+        } != null
```

Comment:
I believe `any()` can be used here: `return any { (dayIndex, date) -> dayIndex == sessionListDay && date == currentDate }`. Please verify.